syz-manager/html.go

```diff
@@ -447,8 +466,45 @@ func (mgr *Manager) httpInput(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "can't find the input", http.StatusInternalServerError)
 		return
 	}
-	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
-	w.Write(inp.Prog)
+	if r.FormValue("raw") == "1" {
```

Comment:
All the code under "if" sounds like a one more function.
It'll be easier to read and easier to test.